package.json

```diff
@@ -52,8 +51,7 @@
   },
   "peerDependencies": {
     "@types/puppeteer": "^0.13.9",
-    "@types/selenium-webdriver": "^3.0.8",
-    "selenium-webdriver": "^3.6.0",
+    "selenium-webdriver": "^4.0.0-alpha.1",
```

Comment:
any breaking changes from ^3 that might cause issues?
I actually don't know of any selenium user as of now so it shouldn't be a problem, but can be good to know and might also indicate that if we had users this would be a major unidriver version

Reply:
Prior to removing the 3.x library, I tried running it and the entire set of tests failed miserably (although, I may have done something wrong).

Therefore, I assumed the tests were written while using 4.x-alpha.1.

Wasn't that the case?

Reply:
might be, all good here then :)

Reply:
👍